apps/storybook/src/react-app-env.d.ts

```diff
@@ -1 +0,0 @@
-/// <reference types="react-scripts" />
```

Comment:
Switching to the file below, which saves us from having to install `react-scripts` in Storybook.